ui/packages/catalog-ui-search/src/main/webapp/js/model/Workspace.js

```diff
@@ -19,6 +19,15 @@ var ColorGenerator = require('js/ColorGenerator');
 var QueryPolling = require('js/QueryPolling');
 require('backbone-associations');
 
+// This is a list of model attributes that if changed we do not want save the workspace for
+const IGNORED_WORKSPACE_ATTRIBUTES = ['result', 'saved', 'metacard.modified', 'id', 'subscribed', 'serverPageIndex'];
+
+/**
+ * Check if any of the `changedAttributes` fall outside of the `IGNORED_WORKSPACE_ATTRIBUTES` list.
+ * @param {model} model 
+ */
+const workspaceShouldBeResaved = (model) => model && _.intersection(Object.keys(model.changedAttributes()), IGNORED_WORKSPACE_ATTRIBUTES).length === 0;
```

Comment:
approving it even though I still don't get it.  I trust this has been thoroughly tested and the right eyes are on it.  However, it still seems to me that the length would just have to not equal model.changedAttributes().length.

I am guessing there is some understood precondition that changedAttributes will come one at a time or something, to avoid 'foo' and 'subscribed' both changing and intersection ending up being length 1, and 'foo' being missed.